cpp/CMakeLists.txt

```diff
@@ -315,6 +311,8 @@ set(CUML_LINK_LIBRARIES
   ${CUDA_nvgraph_LIBRARY}
   ${CUMLPRIMS_MG_LIBRARIES})
 
+ message("FAISS: ${CUML_LINK_LIBRARIES}")
```

Comment:
Debug print?

Reply:
Ha! I think you caught that just as I was removing it.